### DISCUSSION TITLE
[V3] Inline component doesn't render Tailwind styles

### DISCUSSION THREAD

**Discussion by ev3rlost** [livewire/livewire#6417] [1. Help] [answered]
I have a simple inline component, displaying a message only when my app is offline. It seems Tailwind styles are not rendered - at all! It's just displaying plain text, although there are styles applied in the html.  And I can't figure out if it's a Livewire or Tailwind issue. Can anyone help?

**Comment by PhiloNL** [comment]
Have you added the path to your Tailwind config? https://tailwindcss.com/docs/content-configuration

**Reply by ev3rlost** [reply]
I have these two settings in my Tailwind content config:

```  
content: [
    "./resources/**/*.blade.php",
    "./resources/**/*.js",
],
```

Tailwind does work with Livewire full-page components and other standard Laravel blade components - it's just the inline components that seem to struggle rendering Tailwind classes.

My simplified Livewire inline component:

```
class Offline extends Component
{
    public function render()
    {
        return <<<'HTML'
        <div wire:offline class="w-full p-8 bg-red-500 text-white text-4xl">
            You have no internet connection.
        </div>
        HTML;
    }
}
```

The inline component is rendered, Tailwind's classes are applied in the HTML, but visually - nothing but the text "You have no internet connection" is displayed.

**Reply by PhiloNL** [reply, marked as answer]
You need to ensure the components are part of your config, otherwise, Tailwind doesn't know about the inline HTML:

```
content: [
    "./app/Livewire/**/*.php",
],
```

**Reply by ev3rlost** [reply]
Thank you so much Philo! Works like a charm now! :)
